openapi/weaviate/Package.md

```diff
@@ -0,0 +1,18 @@
+Connects to the [Weaviate Vector Search Engine API](https://weaviate.io/developers/weaviate/api) from Ballerina.
+
+## Package overview
+The `ballerinax/weaviate` package is a [Ballerina](https://ballerina.io/) connector for Weaviate Vector Search Engine API. The connector provide access to the manipulations of weaviate schema, objects and search vectors based on various criterias. 
+
+### Compatibility
+|                            | Version                       |
+|----------------------------|-------------------------------|
+| Ballerina Language         | 2201.4.1 (Swan Lake Update 4) |
+| Weaviate Vector Search API | 1.18.0                        |
```

Comment:
```suggestion
| Weaviate vector eearch API | 1.18.0                        |
```

Reply:
Addressed in https://github.com/ballerina-platform/openapi-connectors/pull/754/commits/4011e12c5729754ff4de72e28289bb1362275fd0.